packages/utils/test/unit/promiserace.test.ts

```diff
@@ -1,104 +1,83 @@
-  const precutoff = RaceEvent.precutoff;
-  const cutoff = RaceEvent.cutoff;
-  const resolvedatcutoff = RaceEvent.resolvedatcutoff;
-  const pretimeout = RaceEvent.pretimeout;
-  const timeout = RaceEvent.timeout;
-  const resolved = RaceEvent.resolved;
-  const rejected = RaceEvent.rejected;
-
-  // Second item in testcase row i.e. array of numbers represent delay at which promise to be resolved
-  // or rejected (-ve number)
-  // Third item in testcase row is the expected value or error message in string
-  // Last item is expected events
-  const testcases: [string, number[], (string | Error)[], RaceEvent[]][] = [
-    ["all resolve pre-cutoff", [100, 200], ["100", "200"], [resolved, resolved, precutoff]],
-    ["all resolve/reject pre-cutoff", [100, -200], ["100", "-200"], [resolved, rejected, precutoff]],
-    ["all reject pre-cutoff", [-100, -200], ["-100", "-200"], [rejected, rejected, precutoff]],
-    ["all reject pre-timeout", [-1100, -1200], ["-1100", "-1200"], [cutoff, rejected, rejected, pretimeout]],
-    ["race and resolve pre-timeout", [1100, 1200], ["1100", "pending"], [cutoff, resolved, pretimeout]],
-    [
-      "race and resolve/reject pre-timeout",
-      [-1100, 1200, 1300],
-      ["-1100", "1200", "pending"],
-      [cutoff, rejected, resolved, pretimeout],
-    ],
-    [
-      "some resolve pre-cutoff with no race post cutoff",
-      [100, -200, -1100, 1200],
-      ["100", "-200", "pending", "pending"],
-      [resolved, rejected, resolvedatcutoff],
-    ],
+  const testCases: [string, number[], (string | Error)[]][] = [
+    ["all resolve pre-cutoff", [100, 200], ["100", "200"]],
+    ["all resolve/reject pre-cutoff", [100, -200], ["100", "-200"]],
+    ["all reject pre-cutoff", [-100, -200], ["-100", "-200"]],
+    ["all reject pre-timeout", [-1100, -1200], ["-1100", "-1200"]],
+    ["race and resolve pre-timeout", [1100, 1200], ["1100", "aborted"]],
+    ["race and resolve/reject pre-timeout", [-1100, 1200, 1300], ["-1100", "1200", "aborted"]],
+    ["some resolve pre-cutoff with no race post cutoff", [100, -200, -1100, 1200], ["100", "-200", "-1100", "1200"]],
     [
       "some reject pre-cutoff, with race resolution pre-timeout",
       [-100, -200, -1100, 1100, 1200],
-      ["-100", "-200", "-1100", "1100", "pending"],
-      [rejected, rejected, cutoff, rejected, resolved, pretimeout],
-    ],
-    [
-      "some reject pre-cutoff, rest reject pre-timeout",
-      [-100, -200, -1100, -1200],
-      ["-100", "-200", "-1100", "-1200"],
-      [rejected, rejected, cutoff, rejected, rejected, pretimeout],
+      ["-100", "-200", "-1100", "1100", "aborted"],
     ],
+    ["some reject pre-cutoff, rest reject pre-timeout", [-100, -200, -1100, -1200], ["-100", "-200", "-1100", "-1200"]],
     [
       "some resolve/reject pre-cutoff, some resolve/reject pre-timeout but no race beyond cutoff",
       [100, -200, -1100, 1100, 1700, -1700],
-      ["100", "-200", "pending", "pending", "pending", "pending"],
```

Comment:
This seems to a bug, as the timeout is `1500` so at least 1 before that must be resolved.